core/templates/pages/admin-page/roles-tab/admin-roles-tab.directive.ts

```diff
@@ -183,7 +185,7 @@ angular.module('oppia').directive('adminRolesTab', [
                 contributionRights.can_review_translation_for_language_codes);
               voiceoverLanguages = getLanguageDescriptions(
                 contributionRights.can_review_voiceover_for_language_codes);
-              ctrl.result = {
+              ctrl.contributionReviewersResult = {
```

Comment:
Do we need to initialize these values in init method?

Reply:
These are there in the init method (line nos. 334, 335)